------------------------------------------------------------------------
r12008 | fuscated | 2020-03-30 19:14:41 +0200 (Mon, 30 Mar 2020) | 3 lines

- DragScroll: Remove int3, so it might build on non-x86 machines (ticket #935)

> I guess this is useful only for debugging.

Index: src/plugins/contrib/dragscroll/dragscrollcfg.h
===================================================================
--- a/src/plugins/contrib/dragscroll/dragscrollcfg.h	(revision 12007)
+++ b/src/plugins/contrib/dragscroll/dragscrollcfg.h	(revision 12008)
@@ -35,7 +35,7 @@
         wxString GetBitmapBaseName() const;
         void OnApply();
         void OnCancel(){}
-        virtual void InitDialog(){ asm("int3");} /*trap*/
+        virtual void InitDialog() { } /*trap*/
 
         // pointer to owner of the configuration diaglog needed to
         // complete the OnApply/OnCancel EndModal() logic

------------------------------------------------------------------------
